src/jep/convert_p2j.c

```diff
@@ -41,6 +42,9 @@
 
 #define JCHAR_MAX   0xFFFF
 
+static jmethodID classGetMethods = 0;
+static jmethodID classGetModifiers = 0;
+static jmethodID methodGetModifiers = 0;
```

Comment:
We've got `classGetMethods`  in pyjobject.c, `classGetModifiers` in pyjclass.c, and `methodGetModifiers` in pyjmethod.c.  We need to consolidate to....somewhere.  Now that I look at it we have `Class.getComponentType()` in jep_util.c and pyjarray.c too.  `Class.getParameterTypes()` is in pyjconstructor.c and pyjmethod.c.

Doesn't need to be fixed here.  @bsteffensmeier, we should try to consolidate those.  Maybe we rip more out of jep_util.c and make a jep_init.c or jep_cache.c or jep_reflection.c.  It's not a huge hit since we cache the jmethodIDs but it would still be nice to have the reusable ones in one location.